src/components/AreaSelection.tsx

```diff
@@ -96,7 +98,9 @@ const AreaSelection: React.FC<DropdownBoxProps> = ({
             tagName={t("FACILITATORS.STATE")}
             selectedCategories={selectedState}
             onCategoryChange={handleStateChangeWrapper}
-            overall={isCenterSelection ? false : true}
+            overall={inModal ? false : true}
```

Comment:
@AkshataKatwal16 inModal is already a boolean right?

Reply:
yes